examples/bigcommerce/.vscode/settings.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
I think we should drop this file since this only applies to specific extensions (e.g., cSpell) that users of the example probably won't have. If you want to have these options for yourself, you can set them in the user settings instead.